src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search"
+      action="#"
+      data-qa="big"
+    >
+      <lable class="search__field search__field--big">
```

Comment:
You can move styles from search__field--big to search__field and it will be default input so then you will need modifier only for small input.